micromasters/settings.py

```diff
@@ -487,6 +487,7 @@ def get_var(name, default):
 CELERY_TIMEZONE = 'UTC'
 
 # Elasticsearch
+DEFAULT_ES_PAGE_SIZE = get_var('DEFAULT_ES_PAGE_SIZE', 10)
```

Comment:
Can you rename this to `ELASTICSEARCH_DEFAULT_PAGE_SIZE` to be consistent with the other `ELASTICSEARCH` environment variables?